src/org/camunda/latera/bss/connectors/hid/Hydra.groovy

```diff
@@ -155,22 +152,5 @@ class Hydra implements Ref, Message, DataType, AddParam, Good, Document, Contrac
       num_N_FIRM_ID: firmId
     ])
   }
-
-  Version getVersion() {
```

Comment:
Please don't remove this method and version field because it's quite useful feature. Remove only checks for `< 5.1.2` and `< 5` version because current changes require 5.1.3 and these comparisons don't make any sense now.

Reply:
It's may be useful, but we do not store code, that we don't use. And we don't need to check Hydra version, because we know what version we use with this code.

Reply:
What if you want to save backward compatibility with 5.1.3 then 5.1.4 version of Hydra will be released with another set of breaking changes?

Reply:
After discussion was decided that Hydra and REST API versions in camunda-ext will be fixed.